ci/Jenkinsfile.android

```diff
@@ -83,7 +84,9 @@ pipeline {
         stage('Build') { stages {
           stage('JSBundle') {
             steps {
-              script { cmn.nix.build(attr: 'targets.mobile.jsbundle') }
+              script {
+                cmn.nix.shell('make jsbundle-android', pure: false)
```

Comment:
@jakubgs Any reason `pure` needs to be false here? We should be able to run it in a pure shell.

Reply:
Good point, I'll fix it in #8652.